packages/react-ui/src/theme/sections/components/navigation.js

```diff
@@ -152,6 +152,12 @@ export const navigationOverrides = {
   MuiLink: {
     defaultProps: {
       underline: 'hover'
+    },
+
+    styleOverrides: {
+      root: ({ theme }) => ({
```

Comment:
do you need to deconstruct `theme` if is not used?

Reply:
Right, done